zio-schema/shared/src/main/scala/zio/schema/Schema.scala

```diff
@@ -843,50 +849,102 @@ object Schema {
   }
 
   final case class Sequence[Col[_], A](schemaA: Schema[A], fromChunk: Chunk[A] => Col[A], toChunk: Col[A] => Chunk[A])
-      extends Schema[Col[A]]
+      extends Schema[Col[A]] {
+    def defaultValue: Either[String, Col[A]] = schemaA.defaultValue.map(fromChunk.compose(Chunk(_)))
+  }
 
-  sealed case class Enumeration(structure: ListMap[String, Schema[_]]) extends Schema[(String, _)]
+  sealed case class Enumeration(structure: ListMap[String, Schema[_]]) extends Schema[(String, _)] {
+
+    def defaultValue: Either[String, (String, _)] =
+      if (structure.isEmpty)
+        Left(s"cannot access default value for enumeration with no members")
+      else
+        structure.head._2.defaultValue.map((structure.head._1, _))
```

Comment:
I think in this case we should require a default case be specified when the `Enumeration` is constructed. So maybe change to
```scala
final case class Enumeration(structure: ListMap[String,Schema[_]], defaultCase: Option[String] = None) extends Schema[(String,_)] {
   def defaultValue: Either[String,(String,_)] = 
      defaultCase match {
         case Some(id) => structure.get(id).map(_.defaultValue.map(id -> _)).getOrElse(Left("invalid default case"))
         case None => Left("no default case is defined")
      }